src/store/getters.ts

```diff
@@ -724,11 +724,11 @@ export const getNameIsSlashed = (state: StateIF): boolean => {
 
 export const getShowCorpNum = (state: StateIF): CorpNumRequests.COLIN | CorpNumRequests.MRAS | false => {
```

Comment:
In a future incremental PR, I'll tackle this function, which may be obsolete because we now prompt/search for corp num on the first page instead of the last page.